prompter.ts

```diff
@@ -83,16 +83,20 @@ export class Prompter implements IPrompter {
 		}).future<string>()();
 	}
 
-	public getString(prompt: string, defaultAction?: () => string): IFuture<string> {
+	public getString(prompt: string, options?: IPrompterOptions): IFuture<string> {
 		return (() => {
 			let schema: IPromptSchema = {
 				message: prompt,
 				type: "input",
-				name: "inputString"
+				name: "inputString",
+				validate: (value: any) => {
+					let allowEmpty = options === undefined || options.allowEmpty === undefined ? true : options.allowEmpty;
+					return (!allowEmpty && !value) ? `${prompt} must be non-empty` : true;
```

Comment:
Even better:
```TypeScript
let doesNotAllowEmpty = options && options.allowEmpty;
return (doesNotAllowEmpty && !value) ? `${prompt} must be non-empty` : true;
```

But why the return of validate method is string in one case and boolean in the other?